test/include_tree/include_args_to_hash_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class IncludeTree
+      class IncludeArgsToHashTest < MiniTest::Test
```

Comment:
This should actually be inside `module Parsing`, and then you wouldn't need the fully qualified class name.